content/authors/alvan-wazecky/index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Alvan Wazecky
+type: authors
+images:
+  - url: /engineering-education/authors/alvan-wazecky/avatar.jpg 
```

Comment:
Change "avatar.jpg" to "avatar.png" because the new image path is png.

Reply:
Done